tests/service/Common.fs

```diff
@@ -118,6 +120,7 @@ let mkProjectCommandLineArgsSilent (dllName, fileNames) =
         yield "--target:library" 
         for x in fileNames do 
             yield x
+        System.Diagnostics.Debugger.Break()
```

Comment:
What's this?

Reply:
carelessly left over rubbish from debugging.